x-pack/legacy/plugins/index_management/public/app/components/mappings_editor/components/document_fields/fields/edit_field/edit_field_header_form.tsx

```diff
@@ -107,6 +114,19 @@ export const EditFieldHeaderForm = React.memo(({ type, defaultValue, isMultiFiel
             </EuiFlexItem>
           )}
         </EuiFlexGroup>
+
+        {isMultiField ? (
+          <>
+            <EuiSpacer size="s" />
+            <EuiText size="s" color="subdued">
+              {i18n.translate('xpack.idxMgmt.mappingsEditor.multiFieldsDescription', {
+                defaultMessage:
+                  'Multi-fields are useful to index the same field in different ways.',
```

Comment:
I love this help text. How about we add it to the "Add multi-field" button as a tooltip, too?

Reply:
@cjcenizal I added this to the "Add multi-field" link. WDYT?

It seems like it makes more sense here, rather than the button, since the user would have had to click the link first. (Btw ignore this comment if that is what you originally meant 😄 )

<img width="378" alt="Screen Shot 2019-12-16 at 1 31 34 PM" src="https://user-images.githubusercontent.com/5226211/70932981-f9bd9580-2008-11ea-8126-9d9de105e0c1.png">